TicTacToeApp/ViewController.swift

```diff
@@ -21,25 +20,45 @@ public class ViewController: UIViewController {
     // Dispose of any resources that can be recreated.
   }
 
-  @IBOutlet public var boardButtons: [UIButton]!
-  @IBOutlet weak var resetButton: UIButton!
-
   @IBAction public func resetBoard() {
```

Comment:
I think I make this comment elsewhere, but this function seems to do more than just "reseting the board."